### ISSUE TITLE
Accordions - Role not inside the required context

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#5106]
## Problem

SiteImprove is flagging our accordions with "Role not inside the required context" related to the `role=tab` attribute.

>Make sure the element is contained by another element that has the required context role.

Lighthouse is also flagging this and pointing to these:

https://web.dev/aria-required-parent
https://web.dev/aria-required-children

## Proposed Solution

- Investigate why it is being flagged given that we have `role=tablist` wrapper. Too much separation in nesting divs perhaps?

**Comment by briand44** [comment]
Can we remove this card in blocked/waiting now that we have an issue for this? https://github.com/orgs/uiowa/projects/1#card-76793651

**Comment by briand44** [comment]
Thanks for creating this by the way!

**Comment by joewhitsitt** [comment]
Ah.. I thought it looked familiar. Yes

**Comment by briand44** [comment]
Example

https://teachonline.uiowa.edu/resources/new-semester-icon-setup
This was mentioned in the #redesign-teachonline channel

**Comment by joewhitsitt** [comment]
Semi related for v2: https://github.com/uiowa/uiowa/issues/3567#issuecomment-1098081334

**Comment by briand44** [comment]
>Siteimprove reports the inverse issue as well: according to the report, the TABS in that section are not associated with the TABLIST. I’m also seeing this issue in other sites, so there may be something in the accordion widget that wants adjustment. Stay tuned.

Not sure if this is helpful or not but something TM mentioned in relation to this issue.

**Comment by briand44** [comment]
I reached out to TM to see if he has any recommendations to address this.

**Comment by GaryRidgway** [comment]
@briand44 has there been any response from TM on this? I would like to start it but I believe it is blocked until he responds?

**Comment by e-marie-w** [comment]
I started poking at this. Not sure yet if this is the problem, but https://www.w3.org/TR/wai-aria-practices-1.1/#wai-aria-roles-states-and-properties-20 and https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role#example both mention/use the additional `role=tabpanel` alongside `role=tab` and `role=tablist`

**Comment by briand44** [comment]
>has there been any response from TM on this?

No

>I would like to start it but I believe it is blocked until he responds?

I think we could dig into it and see if there is something we find that could improve this from an accessibility standpoint.

**Comment by joewhitsitt** [comment]
Though this is still an issue, closing in favor of #4004
